cloudapi/client.go

```diff
@@ -70,14 +71,15 @@ func (c *Client) NewRequest(method, url string, data interface{}) (*http.Request
 		buf = bytes.NewBuffer(b)
 	}
 
-	req, err := http.NewRequest(method, url, buf)
+	req, err := http.NewRequest(method, url, buf) //nolint:noctx // the user can add this
```

Comment:
```suggestion
	req, err := http.NewRequest(method, url, buf) //nolint:noctx // the user can add this
```
What do you mean by _the user_? It means in the sense of the caller of the function?

Reply:
Yeah -this is basically a helping method on top of http.NewRequest.